ingeniamotion/pdo.py

```diff
@@ -166,6 +166,7 @@ class ProcessDataThread(threading.Thread):
         ETHERCAT_PDO_WATCHDOG = "processdata"
         PDO_WATCHDOG_INCREMENT_FACTOR = 1.5
         SECONDS_TO_MS_CONVERSION_FACTOR = 1000
+        TIME_SLEEP_PRECISION = 0.016
```

Comment:
Where does this value come from?

Reply:
Chek the jira issue.

Reply:
I'd add a comment with a link to the stackoverflow page.